apps/testingaccessibility/src/components/product-tiers.tsx

```diff
@@ -14,35 +15,37 @@ export const PricingTiers: React.FC<CommerceProps> = ({
   const {redeemableCoupon, RedeemDialogForCoupon} = useCoupon(couponFromCode)
   const purchasedProductIds = purchases.map((purchase) => purchase.productId)
   return (
-    <div className="lg:flex grid lg:gap-5 gap-40">
-      {redeemableCoupon ? <RedeemDialogForCoupon /> : null}
-      {products?.map((product, i) => {
-        const isFirst = i === 0
-        const isLast = i === products.length - 1
-        const isPro = !isFirst && !isLast
+    <PriceCheckProvider purchasedProductIds={purchasedProductIds}>
```

Comment:
pass in the ids of their purchased products to reference for the downgrade check